4_context_manager/my_context_manager.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+
+class my_context:
```

Comment:
remember pep 8?
https://peps.python.org/pep-0008/#class-names

you should always follow those recommendations/rules.